cookbooks/aws-parallelcluster-platform/recipes/install/disable_services.rb

```diff
@@ -27,3 +27,13 @@
 service 'log4j-cve-2021-44228-hotpatch' do
   action %i(disable stop mask)
 end unless on_docker?
+
+# Necessary on Ubuntu and Amazon Linux 2
+service 'cups' do
+  action %i(disable stop mask)
+end unless on_docker?
+
+# Necessary on Ubuntu 22
```

Comment:
Wont this run on all of the OSSes? Would this give us any failures for `cups or wpa_supplicant not found` for other Osses?

Reply:
I expect not to cause failures because the disablement should be resilient if service is not found, but waiting for tests to finish to verify

Reply:
In any case I agree that the best practice is to split the recipe in OS-specific recipes.